libr/io/io.c

```diff
@@ -226,6 +226,19 @@ R_API bool r_io_vwrite_at(RIO *io, ut64 vaddr, const ut8* buf, int len) {
 	return r_io_bank_write_at (io, io->bank, vaddr, buf, len);
 }
 
+R_API bool r_io_vwrite_to_overlay_at(RIO *io, ut64 vaddr, const ut8 *buf, int len) {
+	r_return_val_if_fail (io && buf && len > 0, false);
+	if ((UT64_MAX - (len - 1)) < vaddr) {
+		int _len = UT64_MAX - vaddr + 1;
```

Comment:
```suggestion
		int olen = UT64_MAX - vaddr + 1;
```

better olen than _len but meh

Reply:
olen implies old len, doesn't it

Reply:
Or original_length